src/App.jsx

```diff
@@ -14,45 +17,130 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [listOfGoods, setListOfGoods] = useState([...goodsFromServer]);
+  const [targetInnerText, setTargetInnerText] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
+
+  const sortList = (list, type) => {
```

Comment:
You use the sort method right on function parameters. It is a mutating method. mutate function params is bad practice. let's fix it